integration/storagetest/logtests.go

```diff
@@ -485,21 +489,23 @@ func (*logTests) TestDequeueLeaves(ctx context.Context, t *testing.T, s storage.
 				t.Fatalf("UpdateSequencedLeaves(): %v", err)
 			}
 			return nil
-		})
+		}); err != nil {
+		t.Fatalf("Could not dequeue the expected number of leaves: %v", err)
 	}
 
-	{
-		// If we dequeue again then we should now get nothing
-		runLogTX(s, tree, t, func(ctx context.Context, tx3 storage.LogTreeTX) error {
+	// If we dequeue again then we should now get nothing
+	if err := s.ReadWriteTransaction(cctx, tree,
+		func(ctx context.Context, tx3 storage.LogTreeTX) error {
 			leaves3, err := tx3.DequeueLeaves(ctx, 99, fakeDequeueCutoffTime)
 			if err != nil {
 				t.Fatalf("Failed to dequeue leaves (second time): %v", err)
 			}
 			if len(leaves3) != 0 {
-				t.Fatalf("Dequeued %d leaves but expected to get none", len(leaves3))
+				return status.Errorf(codes.Aborted, "Dequeued %d leaves but expected to get none", len(leaves3))
 			}
 			return nil
-		})
+		}); err != nil {
+		t.Fatalf("Could not dequeue the expected number of leaves: %v", err)
```

Comment:
Maybe add a (got, want) for the number of leaves?

Reply:
Unfortunately, this information is not available in this block. I'll add the err details via `t.Logf`

Reply:
done